neuralforecast/models/nbeatsx.py

```diff
@@ -80,6 +80,27 @@ def forward(self, theta: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
         return backcast, forecast
 
 
+class ExogenousBasis(nn.Module):
+    # Reference: https://github.com/cchallu/nbeatsx
+    def __init__(self, forecast_size: int):
+        super().__init__()
+        self.forecast_size = forecast_size
+
+    def forward(
+        self, theta: torch.Tensor, insample_x: torch.Tensor
```

Comment:
I am thinking about renaming `insample_x` to `futr_exog`
- To match names in other models.
- To avoid confusion on the nature of the exogenous (hist_exog vs futr_exog).